web/html/src/manager/minion/ansible/schedule-playbook.tsx

```diff
@@ -64,7 +64,10 @@ export default function SchedulePlaybook({ playbook, onBack }: SchedulePlaybookP
   }, [playbook, localTime]);
 
   useEffect(() => {
-    setDatetime(Utils.dateWithTimezone(localTime || ""));
+    // TODO: localTime not set in jest
+    if (localTime) {
+      setDatetime(Utils.dateWithTimezone(localTime || ""));
```

Comment:
I think the datetime globals (`window.localTime`, `window.timeZone`, `window.global_user_localization`) will need to be added to the global context, at least until https://github.com/SUSE/spacewalk/issues/15077 gets discussed and resolved. Other globals should not be necessary as far as I can see.